src/software/ai/evaluation/time_to_travel.h

```diff
@@ -0,0 +1,24 @@
+#pragma once
+
+#include "software/time/duration.h"
+
+/**
+ * Calculate minimum time it would take for an robot to move a set distance
+ *
+ * The calculations are done in 1D to simplify calculations and be as performant as
+ * possible
+ *
+ * @param distance The distance which the robot is moving
+ * @param max_velocity The maximum linear velocity the robot can travel at (m/s)
```

Comment:
i would reword this to make it more general, since youve used this method with a rad/s input in `robot::getTimeToOrientation`

Reply:
Updated the comments to use general units